base/stats.go

```diff
@@ -16,13 +52,39 @@ const (
 //to find valid property names
 //
 func init() {
-	StatsExpvars.Add("changesFeeds_total", 0)
-	StatsExpvars.Add("changesFeeds_active", 0)
-	StatsExpvars.Add("requests_total", 0)
-	StatsExpvars.Add("requests_active", 0)
-	StatsExpvars.Add("revisionCache_hits", 0)
-	StatsExpvars.Add("revisionCache_misses", 0)
+	for _, stat := range globalStatNames {
+		StatsExpvars.Add(stat, 0)
+	}
 	TimingExpvars = NewSequenceTimingExpvar(KTimingExpvarFrequency, KTimingExpvarVbNo, "st")
 	StatsExpvars.Set("sequenceTiming", TimingExpvars)
+	InitDbStats("_server")
+
+	ticker := time.NewTicker(statLogInterval)
+	go func () {
+		for {
+			<-ticker.C
+			Statsf(KeyAll, StatsExpvars.String())
```

Comment:
We probably need to think about handling db.Close, and no longer log stats for a database after its been closed.  Otherwise it would require a SG restart to remove db logging.

Reply:
Agreed, but db.Close is only going to matter to that one db, right? We still want to keep logging stats generally, just need a way to remove those expvars.